feature-tests/Cargo.toml

```diff
@@ -12,7 +12,7 @@ version.workspace = true
 release = false
 
 [dependencies]
-delta_kernel = { path = "../kernel", features = ["arrow_53"] }
+delta_kernel = { path = "../kernel", features = ["arrow_54"] }
```

Comment:
why not arrow_55?

Reply:
all of our tests/examples/etc. seemed to be on arrow_53 (lowest supported version) so just kept it that way. could equivalently bump everyone up?